egress/egress-only-tracking.yaml

```diff
@@ -0,0 +1,74 @@
+components:
+  schemas:
+    EgressOnlyTacking:
+      description: |-
+        The list of structures containing Egress offset and it's length to be placed in the packet.
```

Comment:
"One specific egress tracking specification for a  specified port. An application which supports a single or limited number of egress tracking specifications for a single port should return an error if the number of egress tracking specifications for a specific port or in total exceeds its capabilities. "  --> Note : All these suggestions are based on my high level understanding and should be adjusted based on what the user can expect from this feature.

Reply:
Updated.